Index: cdparanoia/cdparanoia.1.jp
===================================================================
--- cdparanoia.orig/cdparanoia.1.jp	2009-11-22 15:52:07.445293755 +0900
+++ cdparanoia/cdparanoia.1.jp	2009-11-22 15:52:28.837295926 +0900
@@ -151,7 +151,7 @@
 .TP
 .B \-Z --disable-paranoia
 データ照合と訂正機能を
-.b 全て
+.B 全て
 無効にします。-Z オプションを用いると、cdparanoia は
 オーバーラップの設定が 0 である cdda2wav と全く同じようにデータの
 読み取りを行います。
